toolchain/lower/function_context.cpp

```diff
@@ -55,7 +57,24 @@ auto FunctionContext::TryToReuseBlock(SemIR::InstBlockId block_id,
 }
 
 auto FunctionContext::LowerBlockContents(SemIR::InstBlockId block_id) -> void {
+  auto crash_inst_id = SemIR::InstId::None;
+
+  // On crash, report the instruction we were lowering.
+  PrettyStackTraceFunction stack_trace_entry([&](llvm::raw_ostream& output) {
+    SemIR::DiagnosticLocConverter converter(
+        file_context_->context().tree_and_subtrees_getters(), &sem_ir());
+    auto [imports, converted] =
+        converter.Convert(SemIR::LocId(crash_inst_id), /*is_token_only=*/false);
+    converted.loc.FormatLocation(output);
+    // TODO: Format SemIR for the instruction we were lowering?
+    output << "lowering "
```

Comment:
I know "filename" is lowercased, but I've been wondering if it should be capital-cased because "Program arguments" is capital-cased and we don't control it (actually, filename is a bit of an exception)